Services/JobRunner/Jobs/DataXDeployJob.cs

```diff
@@ -2,70 +2,70 @@
 // Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License
 // *********************************************************************
-
-using ScenarioTester;
-using System;
-using DataX.ServerScenarios;
-using System.Threading.Tasks;
+
+using ScenarioTester;
+using System;
+using DataX.ServerScenarios;
+using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using System.Collections.Generic;
 using System.Linq;
 using DataX.Utilities.KeyVault;
 using DataX.Utility.Blob;
 
-namespace JobRunner.Jobs
-{
-    /// <summary>
-    /// Runs through a steel thread scenario for a DataX every 10 minutes to ensure DataX E2E works for saving and deploying a job.
-    /// </summary>
-    public class DataXDeployJob : IJob
-    {
-        private readonly ScenarioDescription _scenario;
-        private readonly AppConfig _config;
-        private readonly ILogger _logger;
-        private readonly int _scenarioCount = 1;
-
-        public DataXDeployJob(AppConfig config, ILogger logger)
-        {
-            _config = config;
-            _logger = logger;
-            
+namespace JobRunner.Jobs
+{
+    /// <summary>
+    /// Runs through a steel thread scenario for a DataX every 10 minutes to ensure DataX E2E works for saving and deploying a job.

```

Comment:
where is the 10 configured?  Might want to remove it from comments and bring it up as a const in the code

Reply:
Updated